### PR TITLE
HOCS-1220 - Work stack changes for WCS

### PR DESCRIPTION
**Pull request by hubert-raich**

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/info/api/dto/ConfigurationDto.java [hunk 1]
```diff
@@ -32,9 +32,8 @@ public static ConfigurationDto from(Configuration configuration) {
         String documentLabelsString = configuration.getDocumentLabels();
         List<String> documentLabels = StringUtils.isBlank(documentLabelsString) ? null : new ArrayList<>(Arrays.asList(documentLabelsString.split(",")));
 
-        String workstackColumnsString = configuration.getWorkstackColumns();
-        List<String> workstackColumns = StringUtils.isBlank(workstackColumnsString) ? null : new ArrayList<>(Arrays.asList(workstackColumnsString.split(",")));
 
+        List<WorkstackColumnDto> workstackColumns = configuration.getWorkstackColumns().stream().map(WorkstackColumnDto::from).collect(Collectors.toList());
         List<SearchFieldDto> searchFieldDtos = configuration.getSearchFields().stream().map(SearchFieldDto::from).collect(Collectors.toList());
 
         return new ConfigurationDto(
```
Comment (lewismuirhead) [inline src/main/java/uk/gov/digital/ho/hocs/info/api/dto/ConfigurationDto.java]: I think there should be a test file for this method

File: src/main/java/uk/gov/digital/ho/hocs/info/api/dto/WorkstackColumnDto.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @JsonProperty("dataValueKey")
+    private String dataValueKey;
+
+    @JsonProperty("isFilterable")
+    private boolean filterable;
+
+    @JsonProperty("headerClassName")
+    private String headerClassName;
+
+    public static WorkstackColumnDto from(WorkstackColumn column) {
```
Comment (lewismuirhead) [inline src/main/java/uk/gov/digital/ho/hocs/info/api/dto/WorkstackColumnDto.java]: I think there should be a test file for this method

File: src/main/java/uk/gov/digital/ho/hocs/info/api/dto/ConfigurationDto.java (hunk 1 above)
Comment (hubert-raich) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/dto/ConfigurationDto.java]: Done

File: src/main/java/uk/gov/digital/ho/hocs/info/api/dto/WorkstackColumnDto.java (hunk 2 above)
Comment (hubert-raich) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/dto/WorkstackColumnDto.java]: Done

**Pull request merged**